src/components/blocks/InformationBlock/InformationBlock.test.tsx

```diff
@@ -0,0 +1,95 @@
+import { render, screen } from '@testing-library/react';
+import InformationBlock from './InformationBlock';
+import type { InformationBlockProps } from './InformationBlock';
+
+describe('InformationBlock', () => {
+  describe("with a 'main' variant", () => {
```

Comment:
There is no need to wrap it with `describe`
`with a 'main' variant` can be a title of `it` test case
Also the title of the test case should contain a verb which describes how it should work: `renders with main variant`
And since we're testing `variant=main` we need to test if the `main` class is assigned successfully, not the text inside